crates/autopilot/src/driver_model.rs

```diff
@@ -0,0 +1,124 @@
+//! Types for communicating with drivers.
+
+pub mod quote {
+    use model::u256_decimal;
+    use primitive_types::{H160, U256};
+    use serde::{Deserialize, Serialize};
+
+    #[derive(Clone, Debug, Default, Deserialize, Serialize)]
+    #[serde(rename_all = "camelCase")]
+    pub struct Request {
+        pub sell_token: H160,
+        pub buy_token: H160,
+        pub kind: Kind,
+        #[serde(with = "u256_decimal")]
+        pub amount: U256,
+    }
```

Comment:
In order to support "fast" and "optimal" quoting (one where we build calldata and simulate, and one where we don't), I think we should introduce a flag for that:
```suggestion
        quality: "fast" | "optimal",
    }
```

Reply:
Alternatively, we can use a separate `/trade` endpoint for doing a quote that also returns calldata.